portal/home.php

```diff
@@ -33,319 +35,455 @@
+    <meta name="description" content="Developed By sjpadgett@gmail.com">
 
-    <?php if ($GLOBALS['portal_two_payments']) { ?>
-    $("#payment").load("./portal_payment.php", {}, function () {});
-    <?php } ?>
+    <script type="text/javascript">
+        var tab_mode = true;
 
-    $(".generateDoc_download").click(function () {
-        $("#doc_form").submit();
-    });
+        function restoreSession() {
+            //dummy functions so the dlgopen function will work in the patient portal
+            return true;
+        }
 
-    function showProfileModal() {
-        var title = <?php echo xlj('Demographics Legend Red: Charted Values. Blue: Patient Edits'); ?> + ' ';
+        var isPortal = 1;
+    </script>
+    <?php
+    echo "<script>var cpid=" . js_escape($pid) . ";var cuser=" . js_escape($user) . ";var webRoot=" . js_escape($GLOBALS['web_root']) . ";var ptName=" . js_escape($_SESSION['ptName']) . ";var webroot_url = webRoot;</script>";
 
-        var params = {
-            buttons: [
-                {text: <?php echo xlj('Help'); ?>, close: false, style: 'info', id: 'formHelp'},
-                {text: <?php echo xlj('Cancel'); ?>, close: true, style: 'default'},
-                {text: <?php echo xlj('Revert Edits'); ?>, close: false, style: 'danger', id: 'replaceAllButton'},
-                {text: <?php echo xlj('Send for Review'); ?>, close: false, style: 'success', id: 'donePatientButton'}
-                ],
-            onClosed: 'reload',
-            type: 'GET',
-            url: webRoot + '/portal/patient/patientdata?pid=' + encodeURIComponent(cpid) + '&user=' + encodeURIComponent(cuser)
-        };
-        dlgopen('','','modal-xl', 500, '', title, params);
-    }
+    Header::setupHeader(['no_main-theme', 'datetime-picker', 'patientportal-style']);
+    ?>
 
-    function saveProfile() {
-        page.updateModel();
-    }
+    <script type="text/javascript" src="../interface/main/tabs/js/dialog_utils.js?v=<?php echo $v_js_includes; ?>"></script>
+    <link href="<?php echo $GLOBALS['web_root']; ?>/portal/sign/css/signer_modal.css?v=<?php echo $v_js_includes; ?>" rel="stylesheet" type="text/css" />
+
+    <script src="<?php echo $GLOBALS['web_root']; ?>/portal/sign/assets/signature_pad.umd.js?v=<?php echo $v_js_includes; ?>" type="text/javascript"></script>
+    <script src="<?php echo $GLOBALS['web_root']; ?>/portal/sign/assets/signer_api.js?v=<?php echo $v_js_includes; ?>" type="text/javascript"></script>
+
+    <?php if ($GLOBALS['payment_gateway'] == 'Stripe') { ?>
+        <script type="text/javascript" src="https://js.stripe.com/v3/"></script>
+    <?php } ?>
+    <?php if ($GLOBALS['payment_gateway'] == 'AuthorizeNet') {
+        // Must be loaded from their server
+        $script = "https://jstest.authorize.net/v1/Accept.js"; // test script
+        if ($GLOBALS['gateway_mode_production']) {
+            $script = "https://js.authorize.net/v1/Accept.js"; // Production script
```

Comment:
Is there a way to localize this script collection so not collecting from elsewhere?

Reply:
disregard this, just realized this is from prior code.

Reply:
I wasn't happy about this either. If I remember correctly this was required by vendor or at minimum, highly recommended.
I'll take another look into this when I do Square(which is the vendor I wanted to do in the first place!!).